scripts/unit_test.coffee

```diff
@@ -0,0 +1,12 @@
+module.exports = (robot) ->
+
+	robot.respond /unit_test status/i, (msg) ->
+		robot.http("http://vbujenkins.vistaprint.net/job/Studio.JavaScript.UnitTests/lastBuild/api/json")
+  			.get() (err, res, body) ->
+  				data = JSON.parse(body)
+  				message = "\nLATEST UNIT TEST STATUS:\n" + data.fullDisplayName + ': ' + data.result + ''
+  				if(data.result == 'FAILURE')

```

Comment:
I'm not sure why you think this is going to work since heroku is hosted outside the company